src/dsql/BoolNodes.h

```diff
@@ -138,6 +141,41 @@ class ComparativeBoolNode final : public TypedNode<BoolExprNode, ExprNode::TYPE_
 };
 
 
+class InListBoolNode : public TypedNode<BoolExprNode, ExprNode::TYPE_IN_LIST_BOOL>
+{
+	const static UCHAR blrOp = blr_in_list;
```

Comment:
Why make a constant instead of use the blr directly in gen function?

Reply:
It's used in two methods (also in `internalPrint`) and I don't like duplicating things ;-)